### PR TITLE
add tests

### PR DESCRIPTION
**Pull request by kzin**

### CODE REVIEW

File: TGOCMessages/tgocmessage/src/androidTest/java/br/com/tgoc/tgocmessage/TGOCMessageTest.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    @Test
+    public void test_falseIsMediaType() {
+        message = new TGOCMessage();
+
+        assertEquals(message.isMediaMessage, false);
+    }
+
+    @Test
+    public void test_isMediaType() {
+        message = new TGOCMessage(null, null, null, null);
```
Comment (cardosoedgar) [inline TGOCMessages/tgocmessage/src/androidTest/java/br/com/tgoc/tgocmessage/TGOCMessageTest.java]: Media passed as null should stay as a non-media message. Need to change the code to reflect this.

File: TGOCMessages/tgocmessage/src/androidTest/java/br/com/tgoc/tgocmessage/TGOCMessageTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+/**
+ * Created by Rodrigo.Cavalcante on 30/11/16.
+ */
+@RunWith(AndroidJUnit4.class)
+public class TGOCMessageTest {
+
+    TGOCMessage message = null;
```
Comment (cardosoedgar) [inline TGOCMessages/tgocmessage/src/androidTest/java/br/com/tgoc/tgocmessage/TGOCMessageTest.java]: Don't need this `null` here.

File: TGOCMessages/tgocmessage/src/androidTest/java/br/com/tgoc/tgocmessage/TGOCPhotoMediaItemTest.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    Context context;
+
+    @Before
+    public void setup() {
+        context = InstrumentationRegistry.getContext();
+    }
+
+    @Test
+    public void test_nullPhotoFromResources() {
```
Comment (cardosoedgar) [inline TGOCMessages/tgocmessage/src/androidTest/java/br/com/tgoc/tgocmessage/TGOCPhotoMediaItemTest.java]: Testing not null right? gotta change the method name.

File: TGOCMessages/tgocmessage/src/androidTest/java/br/com/tgoc/tgocmessage/TGOCPhotoMediaItemTest.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        assertEquals(photo.getData(), null);
+    }
+
+    @Test
+    public void test_photoView() {
+        TGOCPhotoMediaItem photo = new TGOCPhotoMediaItem(context.getResources(), R.drawable.ic_send_black_24dp);
+
+        assertNotEquals(photo.getView(context), null);
+    }
+}
```
Comment (cardosoedgar) [inline TGOCMessages/tgocmessage/src/androidTest/java/br/com/tgoc/tgocmessage/TGOCPhotoMediaItemTest.java:55]: Missing test of the context being null trying to get the view.

File: TGOCMessages/tgocmessage/src/androidTest/java/br/com/tgoc/tgocmessage/TGOCLocationMediaItemTest.java [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+import static org.junit.Assert.assertNotEquals;
+
+/**
+ * Created by Rodrigo.Cavalcante on 30/11/16.
+ */
+@RunWith(AndroidJUnit4.class)
+public class TGOCLocationMediaItemTest {
+
+    double lat = -1.0, lng = -48.0;
+    LatLng latLng = null;
```
Comment (cardosoedgar) [inline TGOCMessages/tgocmessage/src/androidTest/java/br/com/tgoc/tgocmessage/TGOCLocationMediaItemTest.java]: remove `null` here.

File: TGOCMessages/tgocmessage/src/androidTest/java/br/com/tgoc/tgocmessage/TGOCLocationMediaItemTest.java [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+        assertEquals(media.getData(), latLng);
+    }
+
+    @Test
+    public void test_locationFromLatLng() {
+        TGOCLocationMediaItem media = new TGOCLocationMediaItem(latLng);
+
+        assertEquals(media.getData(), latLng);
+    }
+
```
Comment (cardosoedgar) [inline TGOCMessages/tgocmessage/src/androidTest/java/br/com/tgoc/tgocmessage/TGOCLocationMediaItemTest.java]: need to test passing null `Location` and `LatLng` objects, as well as null context to get the view.

File: TGOCMessages/tgocmessage/src/main/java/br/com/tgoc/tgocmessage/TGOCMessage.java [hunk 7]
```diff
@@ -7,12 +7,12 @@
...
 public class TGOCMessage implements TGOCMessageInterface {
 
-    String senderId;
-    String senderDisplayName = null;
-    Date date;
-    String text;
-    boolean isMediaMessage = false;
-    TGOCMessageMediaInterface media;
+    private String senderId;
+    private String senderDisplayName = null;
```
Comment (cardosoedgar) [inline TGOCMessages/tgocmessage/src/main/java/br/com/tgoc/tgocmessage/TGOCMessage.java]: Remove this `null` here.

**Review by cardosoedgar — DISMISSED** [review]

**Review by cardosoedgar — APPROVED** [review]
👍
